Todo.py

```diff
@@ -0,0 +1,203 @@
+                line = line.split(';')
+                if command[1] == line[0]:
+                    todoType = line
+                    break
+        
+        if todoType != None:
+            embedColor = int(todoType[1], 16)
+        else:
+            embedColor = 0x28a745
+        
+        newEmbed = discord.Embed(description=command[0], url="", color=embedColor)
+        
+        command[2] = command[2].replace(' ', '') #TODO: Use dfind instead ?
+        if command[2] != "false":
+            if command[2].startswith("<@"):
+                user = ctx.guild.get_member(int(command[2][2:-1]))
+            else:
+                user = ctx.guild.get_member_named(command[2])
+            
+            if user != None:
+                newEmbed.add_field(name="Asssigned to", value=user.mention, inline=True)
+        
+        if command[3] != "false":
+            if command[3].startswith("<#"):
+                repostChannel = ctx.guild.get_channel(int(command[3][2:-1]))
+            else:
+                for chan in ctx.guild.channels:
+                    if(chan.name == command[3]):
+                        repostChannel = chan
+        else:
+            repostChannel = None
+        
+        newEmbed.set_footer(text=command[1])
+        if repostChannel != None:
+            publicTodo = await repostChannel.send(embed=newEmbed)
+            newEmbed.add_field(name="Public repost", value=repostChannel.mention + " : " + str(publicTodo.id), inline=True)
+        
+        message = await channel.send(embed=newEmbed)
+        await message.add_reaction(EMOJIS['wastebasket'])
+        await message.add_reaction(EMOJIS['check'])
+        await message.add_reaction(EMOJIS['hourglass'])
+        await ctx.message.delete()
+
+    @todo.command()
+    async def addtype(self, ctx, command):
+        '''Command to add a todo type.'''
+        command = command.split(";")
+
+        if command[1].startswith("#"):
+            command[1] = command[1][1:]
+        if len(command[1]) != 6:
+            if len(command[1]) != 3:
+                await ctx.send('The color must be in hexadecimal, like this **#ccc** or **#ff0000**')
+                return
+            else:
+                command[1] = command[1]+command[1]
+        color = "0x" + command[1]
+
+        with open(TODO_TYPES, "r+") as file:
+            content = file.readlines()
+            for line in content:
+                line = line.split(";")
+                if line[0] == command[0]:
+                    await ctx.send('There is already a type named **'+command[0]+'**')
+                    return
+            
+            file.write('\n' + command[0] + ';' + color)
+        await ctx.send('You added the label "'+command[0]+'", the embed\'s color for this todo type will be : #' + command[1])
+
+    @todo.command()
+    async def removetype(self, ctx, command):
+        '''Command to add a remove type.'''
+        with open(TODO_TYPES, "r") as file:
```

Comment:
Where is `TODO_TYPES` defined at ? Your PR doesn't include the settings.py file so I'm guessing you didn't change it ? I'm a bit lost tbh.

Reply:
I didn't push the changes but I added it (in settings.py)